sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsActions.cs

```diff
@@ -51,7 +51,7 @@ public TextAnalyticsActions()
         /// The set of <see cref="ExtractSummaryAction"/> that will get executed on the input documents.
         /// </summary>
         /// <remarks>
-        /// This property only applies for <see cref="TextAnalyticsClientOptions.ServiceVersion.V3_2_Preview_2"/> and up.
+        /// This property only applies for <see cref="TextAnalyticsClientOptions.ServiceVersion.V3_1"/>, <see cref="TextAnalyticsClientOptions.ServiceVersion.V2022_04_01_Preview"/>, and up.
         /// </remarks>
         public IReadOnlyCollection<ExtractSummaryAction> ExtractSummaryActions { get; set; }
```

Comment:
If the property only applied for `v3.2-preview2` and up, it didn't apply to `v3.1` as that is a lower service version.
If you want to delete the `v3.2-preview2` then I suggest removing it completely and just say:
`This property only applies for <see cref="TextAnalyticsClientOptions.ServiceVersion.V2022_04_01_Preview"/> and up.`

Reply:
I saw you change it. It also applies to other places in the code. 
What would be better, for me to say "this applies to other places in your code" and you do the search and fix? or to put a comment in every single line where a comment applies

Reply:
oh I didn't realize it applies to other areas in the code - I think it would be helpful if you could mention that in the future and then I will know to search :)